tbx/static_src/sass/components/_event.scss

```diff
@@ -0,0 +1,112 @@
+@use "config" as *;
+
+.event-block {
+    display: grid;
+    grid-template-columns: 1fr;
+    background-color: rgba($color--white, 0.04);
+    color: $color--white;
+
+    @include media-query(large) {
+        grid-template-columns: 3fr 2fr;
+        column-gap: $spacer-half;
+        height: 500px;
```

Comment:
Should this be a `min-height` rather than an exact height? If someone adjusts the text size in their browser it might end up overlapping otherwise.

Reply:
Fixed in 01ba3b7bf115fe2b9651aa60dc38f856bb83dbfc